.gitignore

```diff
@@ -187,3 +187,7 @@ fabric.properties
 
 # Android studio 3.1+ serialized cache file
 .idea/caches/build_file_checksums.ser
+
+.idea/RayTracing.iml
+
+.idea/misc.xml
```

Comment:
+1 from me. 
And I would also add other.iml

![image](https://user-images.githubusercontent.com/29587649/81825926-3b456480-9505-11ea-8357-d0132b16e9c9.png)

Reply:
yeah, but is there any .idea file that we absolutely want to keep... ? I mean, they regenerate usually, i'm not a fan of synching the .idea as it alwaaayyyss changes and people can have their own settings

Reply:
@dccote

Reply:
It has been removed properly in one branch but I don't remember which one.

Reply:
We are only keeping shared files in the .idea folder, none of the user files.

Reply:
Take a look at the .gitignore.  There is a section at the bottom.  I am not sure what we should be doing with .iml files.

Reply:
Yeah on the .idea issue. do we make a separate branch to clean it properly ?